code/modules/mob/living/blood.dm

```diff
@@ -38,6 +38,9 @@
 		if(!isnull(blooddeficiency))
 			blooddeficiency.lose_blood(seconds_per_tick)
 
+	// Some effects are halved mid-combat. Not a reagent count, just a modifier
+	var/determined_mod = reagents.has_reagent(/datum/reagent/determination) ? 0.5 : 0
```

Comment:
Should be checking for `has_status_effect(/datum/status_effect/determined)` IMO, otherwise you get it off of a mild scratch